app/parse.py

```diff
@@ -8,8 +15,43 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_single_quote(quote_soup: BeautifulSoup) -> Quote:
+    return Quote(
+        text=quote_soup.select_one(".text").text,
+        author=quote_soup.select_one(".author").text,
+        tags=[i.text for i in quote_soup.select(".tag")]
+    )
+
+
+def parse_page(page_soup: BeautifulSoup) -> list[Quote]:
+    return [parse_single_quote(quote_soap) for quote_soap in page_soup]
+
+
+def get_all_quotes() -> list:
```

Comment:
add additional typehint for list inner objects like list[YOUR_VALUE_TYPE]
for example: list that contains string elements has typehint list[str]